src/scala/scripts/ScalaPBWorker.scala

```diff
@@ -41,7 +46,7 @@ object ScalaPBWorker extends Worker.Interface {
 
     val code = ProtocBridge.runWithGenerators(
       protoc = exec(extractRequestResult.protoc),
-      namedGenerators = namedGeneratorsWithTypes ++ Seq("scala" -> ScalaPbCodeGenerator),
+      namedGenerators = namedGeneratorsWithTypes ++ Seq("scala" -> MainGenerator),
```

Comment:
Can single element be appended with `:+` here?

Reply:
Done